docs/api.md

```diff
@@ -842,6 +842,23 @@ const child = logger.child({foo: 'bar'}, {level: 'debug'})
 child.debug('debug!') // will log as the `level` property set the level to debug
 ```
 
+##### `options.msgPrefix` (String)
+
+The `msgPrefix` property allows you to specify a prefix for every message of the child logger.
+By default, the parent prefix is inherited.
+If the parent already has a prefix, the prefix of the parent and then the child will be displayed.
+
```

Comment:
I think this is also a root logger option, correct? We should have documentation that explains the option for the root logger. I think we are also missing a test that covers the root logger.

Reply:
no, it's child logger option

Reply:
@jsumners Added support for the root logger now